samples/cellular/nrf_cloud_mqtt_multi_service/sample.yaml

```diff
@@ -1,10 +1,18 @@
 sample:
-  name: nRF Cloud MQTT Multi Service Sample
+  name: nRF Cloud Multi Service Sample
 tests:
-  sample.nrf9160.nrf_cloud_mqtt_multi_service:
+  sample.nrf9160.nrf_cloud_multi_service.mqtt:
     build_only: true
     platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
     integration_platforms:
       - nrf9160dk_nrf9160_ns
       - thingy91_nrf9160_ns
     tags: ci_build
+  sample.nrf9160.nrf_cloud_multi_service.coap:
+    build_only: true
+    platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+      - thingy91_nrf9160_ns
+    extra_args: OVERLAY_CONFIG=overlay_coap.conf
+    tags: ci_build
```

Comment:
TODO: add other entries here so other overlays we provide and support will be built by CI (e.g., FMFU)

Reply:
https://nordicsemi.atlassian.net/browse/IRIS-6808